src/models/thelio-mega-r4-n3/README.md

```diff
@@ -0,0 +1,90 @@
+# Thelio Mega (thelio-mega-r4-n3)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Thelio Mega](./img/thelio-mega-r4-n3.webp)
+
+The System76 Thelio Mega is a desktop with the following specifications:
+
+- CPU options
+    - Supports AMD Ryzen Threadripper PRO 7000 series (Zen 4) processors
+        - [7995WX](https://www.amd.com/en/products/specifications/processors.html?product-0=AMD+Ryzen%E2%84%A2+Threadripper%E2%84%A2+PRO+7995WX)
+        - [7985WX](https://www.amd.com/en/products/specifications/processors.html?product-0=AMD+Ryzen%E2%84%A2+Threadripper%E2%84%A2+PRO+7985WX)
+        - [7975WX](https://www.amd.com/en/products/specifications/processors.html?product-0=AMD+Ryzen%E2%84%A2+Threadripper%E2%84%A2+PRO+7975WX)
+        - [7965WX](https://www.amd.com/en/products/specifications/processors.html?product-0=AMD+Ryzen%E2%84%A2+Threadripper%E2%84%A2+PRO+7965WX)
+        - [7955WX](https://www.amd.com/en/products/specifications/processors.html?product-0=AMD+Ryzen%E2%84%A2+Threadripper%E2%84%A2+PRO+7955WX)
```

Comment:
Since this board is TRX50, it also supports the non-Pro 7000 series (7980X, 7970X, 7960X), even though we aren't shipping with those options in Mega. Should those be listed as well?

Reply:
The non-Pro CPUs lock a few other features that we rely on with our shipped configs. I don't know if it's worth opening that can of worms for something we don't offer, unless we were planning to offer it at some point in the future.

Reply:
Good point, I think it's fine the way it is then.